### ISSUE TITLE
Nit: nn.Identity not necessary?

### ISSUE THREAD

**Issue by gsganden** [warner-benjamin/transformer-from-scratch#2]
Nit: It looks to me like `nn.Dropout` accepts `p=0`, so you can just use it for all cases rather than using `nn.Identity` for that case as suggested in https://github.com/warner-benjamin/transformer-from-scratch/blob/attention_block/exercises/attention_block/attention_block.py#L42
